client/src/components/itemList/ItemCard.js

```diff
@@ -0,0 +1,32 @@
+import styled from '@emotion/styled';
+import React from 'react';
+import { ShowItemIcon } from './Icons';
+import PropTypes from 'prop-types';
+
+export const Card = styled.div`
+  width: 100%;
+  height: 45px;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  font-size: 1.1rem;
+  border-bottom: 1px solid #cbcbcb;
+`;
```

Comment:
```suggestion
export const Card = styled.div`
  width: 100%;
  height: 45px;
  display: flex;
  align-items: center;
  justify-content: space-around;
  font-size: 1.1rem;
  border-bottom: 1px solid #cbcbcb;
  & > div {
  flex-basis: 25%;
  text-align: center;
  }
`;
```

I noticed that when your text in column "Register" is a bit longer, it pushes other `flex-item` in the row to the side. The result ist that the checkbox is not aligned with the header anymore. When you apply `flex-basis: 25%` to the `div` child elements, they will all have an equal size.